network-api/networkapi/templates/pages/libraries/detail_page.html

```diff
@@ -0,0 +1,77 @@
+{% extends "pages/libraries/base.html" %}
+{% load i18n wagtailcore_tags wagtailimages_tags %}
+
+{% block library_content %}
+    <div class="tw-grid tw-grid-cols-1 tw-grid-rows-[auto_auto_1fr] medium:tw-grid-cols-[12rem_1fr_14rem] large:tw-grid-cols-[14rem_1fr_16rem] xlarge:tw-grid-cols-[14rem_1fr_16rem] tw-gap-12">
+
+        <div id="breadcrumbs-container" class="medium:tw-col-start-2 medium:tw-col-end-4">
+            {% block breadcrumbs %}{% endblock breadcrumbs %}
+        </div>
+
+        <div id="title-and-meta" class="medium:tw-col-start-2 medium:tw-col-end-4 xlarge:tw-col-end-3 tw-min-w-0">
+            <h1>{{ page.title }}</h1>
+            <div class="tw-flex tw-flex-row tw-text-xs large:tw-text-base tw-text-gray-80 tw-uppercase tw-divide-x tw-divide-gray-20">
+                {% block detail_page_metadata %}{% endblock detail_page_metadata %}
```

Comment:
The metadata needs to be separate because each library needs different kinds of implementations (one might want the date and authors, other the date, authors and taxonomies, for instance).